english/dream/lucidDream.md

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
It looks like maybe you meant this file to be replaced by the other lucid dream file? If so, please remove this file from the pull request by committing a change that removes the file.